[32mINFO    [0m root:verify_test_05.py:13 ********** Verifying HarvestRight Website Opens *********
[32mINFO    [0m root:verify_test_05.py:15 ********** HarvestRight Website Opens Successfully *********
[32mINFO    [0m root:verify_test_05.py:16 ********** Verifying Login *********
[32mINFO    [0m root:verify_test_05.py:19 ********** Login is Successful *********
[32mINFO    [0m root:verify_test_05.py:20 ********** Verifying it opens the latest purchase order *********
[32mINFO    [0m root:verify_test_05.py:27 ********** Latest purchase order opens successfully *********
[32mINFO    [0m root:verify_test_05.py:29 ********** Verifying it opens receive products *********
[32mINFO    [0m root:verify_test_05.py:31 ********** Successfully open receive products *********
[32mINFO    [0m root:verify_test_05.py:32 ********** Verifying quality check is working properly by adding picture *********
[32mINFO    [0m root:verify_test_05.py:43 ********** Pic is uploaded, Quality check is working properly *********
[32mINFO    [0m root:verify_test_05.py:45 ********** Verifying set quantities is working properly *********
[32mINFO    [0m root:verify_test_05.py:47 ********** Set quantities is working properly *********
[32mINFO    [0m root:verify_test_05.py:50 ********** Verifying add done quantities to products *********
[32mINFO    [0m root:verify_test_05.py:56 ********** Successfully added done quantities *********
[32mINFO    [0m root:verify_test_05.py:57 ********** Verifying put in pack *********
[32mINFO    [0m root:verify_test_05.py:59 ********** Put in pack is working properly *********
[32mINFO    [0m root:verify_test_05.py:61 ********** Verifying lot serial number is added to the products *********
[32mINFO    [0m root:verify_test_05.py:76 ********** Verifying it is validated *********
[32mINFO    [0m root:verify_test_05.py:78 ********** Successfully Validated *********
[32mINFO    [0m root:verify_test_05.py:80 ********** Verifying back order created *********
[32mINFO    [0m root:verify_test_05.py:84 ********** Successfully created back order *********
[32mINFO    [0m root:salesflow_functions.py:109 ********** It is in Done State *******